strings/base_types.h

```diff
@@ -87,8 +87,12 @@ WINRT_EXPORT namespace winrt
         template <typename TStringView>
         static constexpr guid parse(TStringView value)
         {
-            value.remove_prefix(std::min(value.find_first_not_of(" {("), v.size()));
-            value.remove_suffix(std::min(value.find_last_not_of(" })"), v.size()));
+            // Handle {} and ()
+            if (value.size() == 38 && ((value[0] == '{' && value[37] == '}') || (value[0] == '(' && value[37] == ')')))
```

Comment:
Cool! I know `{guid}` is common but I'm not sure I've seen `(guid)` very often... What's the motivation behind that?

Reply:
Based on what the .NET version of Guid.Parse supports: https://docs.microsoft.com/en-us/dotnet/api/system.guid.parse?view=net-6.0